packages/viewer/src/lib/interactions/mouseInteractions.ts

```diff
@@ -139,7 +141,7 @@ export class ZoomInteraction extends MouseInteraction {
       const delta = Point.subtract(position, this.currentPosition);
 
       if (this.startPt != null) {
-        api.zoomCameraToPoint(this.startPt, delta.y);
+        this.zoomCameraToPointWithDirectionConfig(this.startPt, -delta.y, api);
```

Comment:
This `-delta.y` is intentional in order to sync up the zoom by mouse drag to be the same as the mouse wheel, in that a mouse-forward drag produces the same result as a mouse wheel forward event.